### PR TITLE
[service.subtitles.subtitool] 1.1.2

### PR DESCRIPTION
**Pull request by KianMax**
Manual Search Added

### CODE REVIEW

File: service.subtitles.subtitool/resources/lib/SubTiTool.py [hunk 1]
```diff
@@ -15,20 +15,29 @@ def search(self, item, t, langs):
...
-        #xbmc.executebuiltin("Notification(Title," + fulllang + ")")
-        url = "http://www.subtitool.com/api/?query=" + self.filename + "&Lang=" + langs
+        #xbmc.executebuiltin("Notification(Title," + item['mansearchstr'] + ")")
+        QueryString = self.filename;
+
+        if item['mansearch']:
+            QueryString = item['mansearchstr'];
+
+        if len(QueryString) < 6:
+            xbmc.executebuiltin("Notification(Title,Search string is too short)")
```
Comment (learningit) [inline service.subtitles.subtitool/resources/lib/SubTiTool.py]: Please move message strings into strings.po. Thanks.

**Comment by KianMax** [comment]
How can I merge two commits into one without opening another PR?

**Comment by Razzeee** [comment]
You have to rebase, then squash the later commit into the previous one. Then force push. 
Look for a tutorial how to do it for the git client your using and maybe make a backup just in case.

**Comment by KianMax** [comment]
Thank you
I think everything is ok now.

**Pull request merged**